src/core/Global.ttslua

```diff
@@ -353,42 +354,89 @@ end
 -- Left-click: print stats, Right-click: reset stats
 function handleStatTrackerClick(_, _, isRightClick)
   if isRightClick then
-    for key, _ in pairs(tokenDrawingStats) do
-      tokenDrawingStats[key] = {}
-    end
+    resetChaosTokenStatTracker()
   else
     local squidKing = "Nobody"
     local maxSquid  = 0
-    local playerColor, playerName
- 
+    local foundAnyStats = false
+
     for key, personalStats in pairs(tokenDrawingStats) do
-      if personalStats ~= {} then
-        if key == "Overall" then
-          playerColor = "White"
-          playerName = "Overall"
-        else
-          playerColor = playmatApi.getPlayerColor(MAT_GUID_TO_COLOR[key])
-          playerName = Player[playerColor].steam_name or playerColor
-
-          local playerSquidCount = personalStats["Auto-fail"] or 0
-          if playerSquidCount > maxSquid then
-            squidKing = playerName
-            maxSquid = playerSquidCount
-          end
+      local playerColor, playerName
+
+      if key == "Overall" then
+        playerColor = "White"
+        playerName = "Overall"
+      else
+        playerColor = playmatApi.getPlayerColor(MAT_GUID_TO_COLOR[key])
+        playerName = Player[playerColor].steam_name or playerColor
+
+        local playerSquidCount = personalStats["Auto-fail"] or 0
```

Comment:
Nit: Your new initialization guarantees a set of 0s for all tokens, so the "or 0" isn't necessary